.github/workflows/integration-test.yml

```diff
@@ -16,6 +16,9 @@ jobs:
           wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/2.3.0/latest/linux/x64/tar/builds/opensearch/dist/opensearch-min-2.3.0-linux-x64.tar.gz
           tar -xzf opensearch-*.tar.gz
           rm -f opensearch-*.tar.gz
+
+      - name: Check the validation of OpenSearch Security Plugin
```

Comment:
This would fail the `integration-test` workflow, what would you think about creating a separate workflow called something like `prerequisites` that could have a task named "Verify there is matching build of security plugin"

Reply:
Thanks for the reply! The reason I set to check it up here because if this one fails the next test of `Download OpenSearch Security Plugin` will also be failed and break the `integration-test`. I was planning to use this test only as the prerequisite for checking the test of `Download OpenSearch Security Plugin`.  Sorry for the confusions anyways. Please correct me if I miss some of the steps.

Reply:
Here is the UI that our users would see on failures, might be useful to have this spelled out more plainly (Less clicks is better!)
![image](https://user-images.githubusercontent.com/2754967/186497587-2ee9e38d-ae9f-497c-8270-3f139bd1f0ac.png)

Reply:
I think I get it now! Ya I will create a new work flow with a better name for running this separately!  Thanks @peternied!